src/databricks/labs/remorph/reconcile/connectors/data_source.py

```diff
@@ -10,14 +10,28 @@
 )
 
 
+class SecretsProvider:
+    @staticmethod
+    def get_secret(ws: WorkspaceClient, scope: str, key: str):
```

Comment:
```suggestion
    def get_secret(scope: str, key: str):
```
Static methods are not mockable

Reply:
I need ws to access dbutils. removed @staticmethod annotation